Classes/Configuration/Flexform/Registry.php

```diff
@@ -85,12 +84,13 @@ class Registry
     ];
 
     /**
-     * Register default actions
+     * Register default actions.
      */
     public function registerDefaultActions(): void
     {
-        if (!is_array($GLOBALS['TYPO3_CONF_VARS']['EXT']['pxa_product_manager']['switchableControllerActions']['items'])) {
-            $GLOBALS['TYPO3_CONF_VARS']['EXT']['pxa_product_manager']['switchableControllerActions']['items'] = [];
+        $scaKey = 'switchableControllerActions';
```

Comment:
Switchable controller actions are depecated in v10 and will be removed in v11. We should replace them.

Reply:
Maybe we can do it when we make changes to controller and actions instead?

Reply:
Sounds like the right thing to do. I'll add info about it to the issue.